pandas/core/strings/object_array.py

```diff
@@ -73,8 +73,9 @@ def _str_map(self, f, na_value=None, dtype: Optional[Dtype] = None):
             arr = np.asarray(arr, dtype=object)  # type: ignore[assignment]
         mask = isna(arr)
         convert = not np.all(mask)
+        ndarr = np.asarray(arr)
```

Comment:
why is this added. not already done in L70-73?

Reply:
good catch, can rename variable there (i think this might have snuck in from another branch as it appears unrelated to index.pyx)